Json - New content: remove object from json array, validate json

tethryus opened the pull request:
Added test cases.

mihaitusa commented:
@tethryus  This commit should contain files related to JSON content and not related to Circle CI

mihaitusa commented on content/io/cloudslang/base/json/remove_object_from_json_array.sl:
```diff
@@ -0,0 +1,57 @@
+#   (c) Copyright 2015 Hewlett-Packard Development Company, L.P.
```
replace with ... (c) Copyright 2016 Hewlett-Packard Enterprise Development Company, L.P. ... in all doc's

mihaitusa commented on content/io/cloudslang/base/json/remove_object_from_json_array.sl:
```diff
@@ -0,0 +1,57 @@
+        import json
+        decoded_json_array = json.loads(json_array)
+        decoded_json_object = json.loads (json_object)
+        index = locals().get('index')
```
Get rid of locals().get('whatever')... is not needed

mihaitusa commented on content/io/cloudslang/base/json/remove_object_from_json_array.sl:
```diff
@@ -0,0 +1,57 @@
+  inputs:
+    - json_array
+    - json_object
+    - index:
```
add here default: None

tethryus replied:
default: None added. thanks for the input.

mihaitusa commented on content/io/cloudslang/base/json/validate_json.sl:
```diff
@@ -0,0 +1,42 @@
+####################################################
+#!!
+#! @description: Validate json input.
+#! @input json_input: JSON from which to retrieve keys - Example: '{"k1": {"k2": {"k3":"v3"}}}'
```
I don't think the description is appropriate here.

mihaitusa commented on test/io/cloudslang/base/json/test_remove_object_from_json_array.sl:
```diff
@@ -0,0 +1,51 @@
+
+        navigate:
+          - SUCCESS: test_equality
+          - FAILURE: CREATEFAILURE
```
replace with CREATE_FAILURE to be more snake_case....

mihaitusa commented:
@tethryus  Please handle the comments so we can merge it!

tethryus commented:
Thanks for pointing the issues out. Addressed all the comment. Thanks for the input on all issues discoevered. Everything should be ok and ready to merge now.
